canopen/pdo/base.py

```diff
@@ -30,13 +30,18 @@ def __iter__(self):
         return iter(self.map)
 
     def __getitem__(self, key):
-        if isinstance(key, int):
+        if isinstance(key, int) and key in list(set(list(range(0x1A00, 0x1A80)) + # By TPDO ID
+                                                    list(range(0x1600, 0x1680)) + # By RPDO ID
+                                                    list(range(0, 80)))):         # By PDO Index
```

Comment:
Hmm, maybe it's better to just check value ranges. It feels a little bit unnecessary to create a large list of valid values each time.

```python
if isinstance(key, int):
    if 0x1A00 < key < 0x1A80 or 0x1600 < key < 0x1680 or 0 < key < 80:
```

Reply:
Done.